messages/notify_test.go

```diff
@@ -15,13 +15,16 @@ func TestNotify(t *testing.T) {
 		{
 			"has-all-params",
 			NewNotify(
+				params.NewStatus(params.AsStateActive),
 				params.NewAspIdentifier(2),
 				params.NewRoutingContext(1),
 				params.NewInfoString("deadbeef"),
 			),
 			[]byte{
 				// Header
 				0x01, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x2c,
+				// Status
+				0x00, 0x0d, 0x00, 0x08, 0x00, 0x01, 0x00, 0x03,
```

Comment:
Since Notify has Status param, the length of Header is no longer correct. Please add 8 bytes to length field (`0x2c => 0x34`).

_this should be applied to other test cases, too._

Reply:
Oops, I'm sorry. Those were so silly mistakes... Fixed at 804b94e020484372471b5a9bbc2da8b590e02762

Reply:
Looks good. Thank you.